mods/parsoid.js

```diff
@@ -14,6 +14,28 @@ var yaml = require('js-yaml');
 var fs = require('fs');
 var spec = yaml.safeLoad(fs.readFileSync(__dirname + '/parsoid.yaml'));
 
+// THIS IS EXPERIMENTAL AND ADDED FOR TESTING PURPOSE!
+// SHOULD BE REWRITTEN WHEN DEPENDENCY TRACKING SYSTEM IS IMPLEMENTED!
+var Purger = require('htcp-purge');
+var Template = require('../lib/reqTemplate');
+var cacheURIs = [
+    // /page/mobile-html/{title}
+    new Template({uri: 'https://rest.wikimedia.org/{domain}/v1/page/mobile-html/{title}'}),
```

Comment:
@gwicke we've chatted with bblack about it and he suggested to purge them too, as he's not sure which URIs are used by varnish - original or rewritten.